pkg/node/compute.go

```diff
@@ -183,6 +186,21 @@ func NewComputeNode(
 		Buffer:         config.LogStreamBufferSize,
 	})
 
+	var localPublisherServer *local.LocalPublisherServer
+	if pub, err := publishers.Get(ctx, models.PublisherLocal); err == nil {
+		ok, err := pub.IsInstalled(ctx)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to check if local publisher is installed")
+		}
+		if ok {
+			log.Ctx(ctx).Info().Msg("**** local publisher is installed and server being started")
+			localPublisherServer = local.NewLocalPublisherServer(ctx, config.LocalPublisher)
+			go localPublisherServer.Start(ctx)
+		}
+	} else {
+		log.Ctx(ctx).Error().Err(err).Msg("local publisher not installed")
+	}
+
```

Comment:
This is a bit difficult to read. Does it make sense to just have the local publisher to be the one running the web server? This allows `IsInstalled()` to return true only if the server is actually running. Right now we are enabling the publisher without knowing if it is running

Reply:
Yes, we can do that, and was actually my initial approach.  I backed it out and I can't remember why, but will put it back.

Reply:
Ah I remember, it was being able to shutdown the server in cleanup as there's no notification to the poublishers that they're being shutdown.